### DISCUSSION TITLE
Nvidia Game Ready Driver may not work!?

### DISCUSSION THREAD

**Discussion by ReflectionRip** [d8ahazard/sd_dreambooth_extension#166] [General]
I was having a really hard time getting this to work, even though I was able to use webui to generate images. After a bunch of messing around I found that Torch wasn't detecting my video card correctly all the time and was failing on training with a very useless error message 'CUDA error: invalid argument'. After much searching and research, I was able to get it to work after I installed the gforce experience and studio driver. The game ready drivers for some reason did not work.

**Comment by d8ahazard** [comment]
I've made a mental note of this, I bet it's the reason for at least one of the open issues we currently have...

**Comment by sgsdxzy** [comment]
I would always recommand installing the driver shipped with the CUDA installer.
